source/app/pages/documents/view.js

```diff
@@ -355,12 +361,17 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
           </div>
           <div className={css.content} ref={contentRef}>
             <DocumentViewer
-              className={cs(
+              className={ENABLE_COMPREHEND_MEDICAL? cs(
```

Comment:
Why do we need this? Isn't it impossible for `tab === 'medical_entities'` to be `true` if the feature is disabled anyway?